src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -159,7 +153,7 @@ const useGroupInfo = () => {
         newData.push(newObj);
       });
     }
-    setNewParticipan(newData);
+    setNewParticipant(newData);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setNewParticipant` is now being used instead of `setNewParticipan`. This is a good change as it corrects the typo in the previous function name.

```diff
- setNewParticipan(newData);
+ setNewParticipant(newData);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->